src/pages/team/index_page.cr

```diff
@@ -0,0 +1,128 @@
+class Team::IndexPage < PageLayout
+  record Member,
+    fullname : String,
+    avatar : String,
+    title : String,
+    github_link : String,
+    twitter_link : String? = nil,
+    mastodon_link : String? = nil
+
+  def content
+    div class: "pt-16 pb-32" do
+      div class: "mx-auto max-w-7xl px-6 text-center lg:px-8" do
+        div class: "mx-auto max-w-2xl" do
+          h2 "Meet the Lucky team", class: "text-3xl font-bold tracking-tight text-gray-900 sm:text-4xl"
+        end
+        ul class: "mx-auto mt-20 px-0 grid max-w-2xl grid-cols-1 gap-x-8 gap-y-16 sm:grid-cols-2 lg:mx-0 lg:max-w-none lg:grid-cols-3", role: "list" do
+          li class: "col-span-full" do
+            render_member(creator)
+          end
+          team_members.each do |member|
+            li do
+              render_member(member)
+            end
+          end
+        end
+      end
+    end
+  end
+
+  def creator
+    Member.new(
+      fullname: "Paul Smith",
+      title: "Creator",
+      avatar: "https://avatars.githubusercontent.com/u/22394?v=4",
+      github_link: "https://github.com/paulcsmith"
+    )
+  end
+
+  def team_members
+    [
+      Member.new(
+        fullname: "Jeremy Woertink",
+        title: "Core Member",
+        avatar: "https://avatars.githubusercontent.com/u/2391?v=4",
+        github_link: "https://github.com/jwoertink",
+        twitter_link: "https://twitter.com/jeremywoertink"
+      ),
+      Member.new(
+        fullname: "Matthew McGarvey",
+        title: "Core Member",
+        avatar: "https://avatars.githubusercontent.com/u/17329408?v=4",
+        github_link: "https://github.com/matthewmcgarvey"
+      ),
+      Member.new(
+        fullname: "Stephen Dolan",
+        title: "Core Member",
+        avatar: "https://avatars.githubusercontent.com/u/6677875?v=4",
+        github_link: "https://github.com/stephendolan",
+        twitter_link: "https://twitter.com/Stephen_Dolan"
+      ),
+      Member.new(
+        fullname: "Edward Loveall",
+        title: "Core Member",
+        avatar: "https://avatars.githubusercontent.com/u/49549?v=4",
+        github_link: "https://github.com/edwardloveall"
+      ),
+      Member.new(
+        fullname: "Michael Wagner",
+        title: "Core Member",
+        avatar: "https://avatars.githubusercontent.com/u/13472976?v=4",
+        github_link: "https://github.com/mdwagner",
+        mastodon_link: "https://mindly.social/@chocolate42"
+      ),
+    ] of Member
+  end
+
+  private def render_member(member : Member)
+    tag "object", data: member.avatar, class: "mx-auto h-56 w-56 rounded-full", type: "image/jpeg", title: member.fullname  do
```

Comment:
For anyone not aware, `<object>` html tags are for loading things with a fallback, so this tries to load the image from github, and if it fails it will load a generic profile image instead.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/object